src/test/java/uk/gov/companieshouse/pscfiling/api/service/impl/TestBaseService.java

```diff
@@ -19,18 +19,17 @@ public class TestBaseService {
 
     //Use to mock out the PscTypeConstants enum class
     @SuppressWarnings("unchecked")
-    static <PscTypeConstants extends Enum<PscTypeConstants>> PscTypeConstants[] addNewEnumValue(
-            final Class<PscTypeConstants> enumPscTypeConstants) {
-        final EnumSet<PscTypeConstants> enumSet = EnumSet.allOf(enumPscTypeConstants);
+    static <PscTypeConstants extends Enum<PscTypeConstants>> PscTypeConstants[] addNewEnumValue() {
+        final EnumSet<PscTypeConstants> enumSet = EnumSet.allOf((Class<PscTypeConstants>) uk.gov.companieshouse.pscfiling.api.model.PscTypeConstants.class);
         final PscTypeConstants[] newValues =
-                (PscTypeConstants[]) Array.newInstance(enumPscTypeConstants, enumSet.size() + 1);
+                (PscTypeConstants[]) Array.newInstance(uk.gov.companieshouse.pscfiling.api.model.PscTypeConstants.class, enumSet.size() + 1);
         int i = 0;
         for (final PscTypeConstants value : enumSet) {
             newValues[i] = value;
             i++;
         }
 
-        final PscTypeConstants newEnumValue = mock(enumPscTypeConstants);
+        final PscTypeConstants newEnumValue = mock((Class<PscTypeConstants>) uk.gov.companieshouse.pscfiling.api.model.PscTypeConstants.class);
```

Comment:
PscTypeConstants is already imported. Do these need the full classpath?

Reply:
It complains if we remove the full classpath, I think this has something to do with the PscTypeConstants on line 22

Reply:
What about just going back to the original method of passing the class into the constructor. Just tried locally and it still works ok

Reply:
done